MaterialComponent.js

```diff
@@ -20,6 +20,12 @@ export default class MaterialComponent extends Component {
     // The final class name given to the dom
     this.classText = "";
   }
+  static uid() {
```

Comment:
Sure. Thought I'd put it there for future-proofing because other elements might need it. But for now it's probably better to keep it in ```Textfield```.